starknet_py/tests/e2e/block_test.py

```diff
@@ -68,17 +65,15 @@ async def test_get_block_with_txs_pending(account):
 
 
 # TODO (#1419): Fix contract redeclaration
-@pytest.mark.skip(reason="Redeclaration occurred")
 @pytest.mark.asyncio
-async def test_get_block_with_txs_latest(
-    account,
-    map_contract_declare_hash,
-):
+async def test_get_block_with_txs_latest(account, map_class_hash):
+    # pylint: disable=unused-argument
+
     blk = await account.client.get_block_with_txs(block_number="latest")
 
     assert isinstance(blk, StarknetBlock)
     assert isinstance(blk.transactions, list)
-    assert blk.transactions[0].hash == map_contract_declare_hash
+    assert blk.transactions[0].hash is not None
```

Comment:
Due to Devnet update and Cairo 1 version of the contract, it is no longer possible to declare the contract at this point. It is intended to update this test in the future - add a declaration of contract, and then there is the confidence that at index 0 there will be 'decalre transaction'